rcl/include/rcl/arguments.h

```diff
@@ -52,6 +55,11 @@ rcl_get_zero_initialized_arguments(void);
  * Successfully parsed remap rules are stored in the order they were given in `argv`.
  * If given arguments `{"__ns:=/foo", "__ns:=/bar"}` then the namespace used by nodes in this
  * process will be `/foo` and not `/bar`.
+ *
+ * The default log level will be parsed as `__log_level:=level`, where `level` is a name representing
+ * one of the log levels in the `RCUTILS_LOG_SEVERITY` enum, e.g. `info`, `debug`, `warn`, not case
+ * sensitive.
```

Comment:
What's the expected behavior if `__log_level:=` is used more than once in command line arguments? Error? Last one wins? First one wins?

Reply:
Last one, documentation added in 0ef1505